site/styles/_variables.scss

```diff
@@ -111,8 +111,8 @@
   --signin-sso-btn-background-hover-color: var(--button-background-hover);
 
   // Carousel
```

Comment:
in terms of rollout, we'd have to think about if we'd want to make changes to the default carousel height for all clients, or leave it the same and remember to adjust for each particular client. It likely would result in better carousel image though making less tall on mobile. more testing required to make sure all elements can fit if they are using all the things and have long film names etc.